README.md

```diff
@@ -1,20 +1,19 @@
 # Oasis Playground
 
-This repository contains dApps running on the Oasis platform for demonstration
-and educational purposes.
+This repository contains an index of notable dApps running on the Oasis
+platform. The index is primarily tailored, but not limited to, technology
+showcases and educational material—hence the name "Playground".
 
-Demo projects are publicly accessible at https://playground.oasis.io.
+The Oasis Playground is publicly accessible at https://playground.oasis.io.
 
 ## Contributing
 
-Developers are encouraged to contribute their applications to this repository.
-Your dApp should be self-contained (backend, contracts, frontend) inside your
-dApp folder. Then, open a pull request and one of the Oasis Protocol
-Foundation members will check it out and get back to you.
+Developers are encouraged to contribute their dApps to the Oasis Playground.
+See [CONTRIBUTING.md](./CONTRIBUTING.md) for details.
 
 ## Disclaimer
```

Comment:
Maybe make this section more prominent, perhaps by bolding the entire paragraph or similar.